src/library_syscall.js

```diff
@@ -1039,7 +1039,7 @@ var SyscallsLibrary = {
   __syscall203__sig: 'ii',
   __syscall203__nothrow: true,
   __syscall203__proxy: false,
-  __syscall203: '__sysicall214',     // setreuid32
```

Comment:
Is this just a typo that has ways been there?     What happens if you try to call this function with such a typo extant?

Reply:
Yes, this was in the tree - you'll probably get an undefined function error?